service/src/main/java/org/cbioportal/service/impl/ExpressionEnrichmentServiceImpl.java

```diff
@@ -146,4 +268,13 @@ private double calculateStandardDeviation(double[] values) {
         }
         return descriptiveStatistics.getStandardDeviation();
     }
+
+    private void validateMolecularProfile(MolecularProfile molecularProfile) throws MolecularProfileNotFoundException {
+        if (molecularProfile.getMolecularAlterationType().equals(MolecularAlterationType.MUTATION_EXTENDED)
```

Comment:
Should STRUCTURAL_VARIANT be placed here (thats a soon to be released type)?  Maybe its safer to list the supported alteration types and not worry about updates to the MolecularAlterationType class.

Reply:
not sure, I was using the existing logic. Let's update it once STRUCTURAL_VARIANT is supported